src/GeositeFramework/plugins/launchpad/main.js

```diff
@@ -54,6 +54,18 @@ define([
                         var saveCode = $(e.target).data('saveCode');
                         self.plugin.turnOff();
                         self.triggerEvent('launchpad:activate-scenario', saveCode);
+                    })
+                    .on('click', 'button.truncate-toggle', function (e) {
+                        var parentEl = $(e.target.parentElement);
+                        parentEl.find('p').toggleClass('truncate-box');
+                        parentEl.find('.truncate-toggle').each(function () {
+                            if ($(this).attr('data-i18n') === 'See More') {
+                                $(this).attr('data-i18n', 'See Less');
+                            } else {
+                                $(this).attr('data-i18n', 'See More');
+                            }
+                            $(this).localize();
```

Comment:
Imagine how one might implement this feature using React. Clicking on the toggle button would update the component state, then we would re-render the component using the new state. 

That is the general design pattern that we are trying to follow with TNC plugins, and that's what I would recommend that we do here. The result should be a clean separation between the state and the template. No DOM manipulation should be necessary.